src/services/completions.ts

```diff
@@ -1543,7 +1543,7 @@ namespace ts.Completions {
 
         function getVariableDeclaration(property: Node): VariableDeclaration | undefined {
             const variableDeclaration = findAncestor(property, (node) => {
-                if (isFunctionLikeDeclaration(node)) {
+                if (isFunctionBlock(node) || isBindingPattern(node)) {

```

Comment:
This doesn’t work for arrow function expressions without braces, e.g. `() => expression/**/`. It seems like there should be a util for this but I can’t find an existing one: `isFunctionBlock(node) || isArrowFunction(node.parent) && node.parent.expression === node`

Reply:
Yes. But I found no node.parent.expression. maybe node.parent.body?
`isFunctionBlock(node) || (isArrowFunction(node.parent) && node.parent.body === node)`

Reply:
Yeah, `body` is what I meant 👍